sections/semantics-tabular-data.include

```diff
@@ -654,7 +654,10 @@
     <dt><a>Contexts in which this element can be used</a>:</dt>
     <dd>As the first element child of a <{table}> element.</dd>
     <dt><a>Content model</a>:</dt>
-    <dd><a>Flow content</a>, but with no descendant <{table}> elements.</dd>
+    <dd>
+      <a>Flow content</a>, excluding the <{main}> element,
+      and no descendant <{table}> elements.
+    </dd>
```

Comment:
````
<caption>
<div>
<main>.......</main>
</div>
</caption>
````
→Is this conforming?


The following is better.
````
<dd>
    <a>Flow content</a>, but with no <{main}>, or <{table}> descendants. 
</dd>
````

Reply:
thanks!